command/elevator.py

```diff
@@ -0,0 +1,45 @@
+import time
+import utils
+
+from subsystem import Elevator
+from robotpy_toolkit_7407.command import SubsystemCommand
+class ZeroElevator(SubsystemCommand[Elevator]):
+
+    def __init__(self, subsystem: Elevator):
+        super().__init__(subsystem)
+        self.subsystem = subsystem
+        self.start_time = None
+
+    def initialize(self) -> None:
+        """
+        Zeros the elevator
+        """
+        # Why was this -0.05 in the old code, I am setting as zero now assuming it was a misconfiguration with the
+        # hardware. If not, please switch back.
+        self.subsystem.motor_extend.set_raw_output(0)
+        self.start_time = time.time()
```

Comment:
this needs to be a negative number in order for the elevator to roll back down to zero. even though this elevator is fixed to the body of the robot, it needs a negative value to be sure its rolling down.